apps/test/unit/templates/rubrics/RubricContainerTest.jsx

```diff
@@ -231,4 +234,60 @@ describe('RubricContainer', () => {
       'Error submitting feedback to student.'
     );
   });
+
+  it('passes down aiUnderstanding and aiConfidence to the LearningGoal', async () => {
+    const mockFetch = sinon.stub(global, 'fetch');
+    const aiEvaluationsMock = [
+      {learning_goal_id: 2, understanding: 2, confidence: 70},
+    ];
+    mockFetch.returns(
+      Promise.resolve(new Response(JSON.stringify(aiEvaluationsMock)))
+    );
+
+    const wrapper = mount(
+      <RubricContainer
+        {...defaultProps}
+        studentLevelInfo={{
+          name: 'Grace Hopper',
+          timeSpent: 305,
+          lastAttempt: '1980-07-31T00:00:00.000Z',
+          attempts: 6,
+        }}
+      />
+    );
+
+    await act(async () => {
```

Comment:
<insert "look who's back.  back again..." as rapped by Eminem>

I know we went back and forth on this but it keeps popping up as the best solution here.  I felt like [this](https://github.com/threepointone/react-act-examples/blob/master/sync.md#promises) was the best explanation of why this works I could find.

Happy to chat about it too.

Reply:
This makes sense to me. Thanks for the link!